dbms/src/Flash/Coprocessor/DAGDriver.cpp

```diff
@@ -110,6 +110,11 @@ try
             dag_context);
         dag_output_stream = std::make_shared<DAGBlockOutputStream>(streams.in->getHeader(), std::move(response_writer));
         copyData(*streams.in, *dag_output_stream);
+        if (dag_context.collect_execution_summaries)
```

Comment:
does it have the same effect as the original one in `UnaryDAGResponseWriter::finishWrite`

Reply:
yes, it does.